buildSrc/src/main/java/org/opensearch/gradle/testclusters/RunTask.java

```diff
@@ -134,12 +141,22 @@ public void beforeStart() {
         }
 
         for (OpenSearchCluster cluster : getClusters()) {
-            cluster.getFirstNode().setHttpPort(String.valueOf(httpPort));
+            // Configure the first node with the default ports first
```

Comment:
I don't see anything special about the first node.  I think we can remove these assignments and the condition on line 153?

Reply:
Agreed, but there is logic in several places that is tied to the notion of the cluster's first node (like [credentials](https://github.com/kartg/OpenSearch/blob/73ce7bbbe40af252873b5fe99dda9c8b9c938022/buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchCluster.java#L529)). Hence my hesitancy to have the port assignment be non-deterministic.